src/canopy-webpack-config.js

```diff
@@ -36,8 +35,12 @@ module.exports = function(name, overridesConfig) {
       entry: `./src/${name}.js`,
       output: {
         filename: `${name}.js`,
-        jsonpFunction: "wpjsonp-"+name,
-        libraryTarget: 'amd',
+        publicPath: '',
+        uniqueName: name,
+        library: {
+          type: 'amd',
```

Comment:
keeping `amd` for now - the `system` output isn't working - looks like more work is needed around the deps we're using